packages/web/src/dom.rs

```diff
@@ -175,3 +188,39 @@ fn walk_event_for_id(event: &web_sys::Event) -> Option<(ElementId, web_sys::Elem
         }
     }
 }
+
+// TODO: Merge with walk_event_for_id
```

Comment:
It looks like this was merged and can be removed?

Reply:
Indeed. The `walk_node_for_id` function is removed.